src/Slider.jsx

```diff
@@ -694,76 +728,142 @@ class Rheostat extends React.Component {
-              role="slider"
-              style={handleStyle}
-              tabIndex={0}
-            />
-          );
-        })}
-        {handlePos.map((node, idx, arr) => {
+            return (
+              <Handle
+                aria-valuemax={this.getMaxValue(idx)}
+                aria-valuemin={this.getMinValue(idx)}
+                aria-valuenow={values[idx]}
+                aria-disabled={disabled}
+                data-handle-key={idx}
+                key={
+                  // eslint-disable-next-line react/no-array-index-key
+                  idx
+                }
+                orientation={orientation}
+                disabled={disabled}
+                onClick={this.killEvent}
+                onKeyDown={disabled ? undefined : this.handleKeydown}
+                onMouseDown={disabled ? undefined : this.startMouseSlide}
+                onTouchStart={disabled ? undefined : this.startTouchSlide}
+                handleRef={this.setHandleNode}
+                role="slider"
+                style={handleStyle}
+                tabIndex={0}
+              />
+            );
+          })}
+        </div>
+        {!!ProgressBar && handlePos.map((node, idx, arr) => {
           if (idx === 0 && arr.length > 1) {
             return null;
           }
-
           return (
             <ProgressBar
-              className="rheostat-progress"
-              key={`progress-bar-${idx}`}
+              key={
+                // eslint-disable-next-line react/no-array-index-key
+                idx
+              }
               style={this.getProgressStyle(idx)}
+              disabled={disabled}
             />
           );
         })}
         {PitComponent && pitPoints.map((n) => {
-          let pitStyle = this.pitStyleCache[n];
-
-          if (!pitStyle) {
-            const pos = algorithm.getPosition(n, min, max);
-            pitStyle = orientation === 'vertical'
-              ? { top: `${pos}%`, position: 'absolute' }
-              : { left: `${pos}%`, position: 'absolute' };
-            this.pitStyleCache[n] = pitStyle;
-          }
+          const pos = algorithm.getPosition(n, min, max);
+          const pitStyle = orientation === VERTICAL
+            ? { top: `${pos}%`, position: 'absolute' }
+            : { left: `${pos}%`, position: 'absolute' };
 
           return (
-            <PitComponent key={`pit-${n}`} style={pitStyle}>{n}</PitComponent>
+            <PitComponent key={n} style={pitStyle}>{n}</PitComponent>
           );
         })}
         {children}
       </div>
     );
   }
 }
+
 Rheostat.propTypes = propTypes;
 Rheostat.defaultProps = defaultProps;
 
-export default Rheostat;
+export default withStyles(({ color, unit, responsive }) => ({
+  rheostat: {
+    position: 'relative',
+    overflow: 'visible',
+  },
+
+  autoAdjustVerticalPosition: {
+    [responsive.largeAndAbove]: {
+      top: 1.5 * unit,
+    },
+  },
+
+  rheostat_vertical: {
```

Comment:
Why are some of these capitalized and not others? `Rheostat_background ` versus `rheostat_vertical `?

Reply:
and a tiny nit, let's have this be `rheostat__vertical` since vertical is a modifier